pom.xml

```diff
@@ -251,6 +253,65 @@
                 <version>${project.version}</version>
             </dependency>
 
+            <!--Ei Analytics solution dependencies-->
+            <dependency>
+                <groupId>org.wso2.analytics.solutions</groupId>
+                <artifactId>wso2sp-solutions.ei-analytics.statschart</artifactId>
+                <version>1.0.0-SNAPSHOT</version>
```

Comment:
Please use ${project.version} instead of `1.0.0-SNAPSHOT` . Please fix in other places as well.

Reply:
Fixed in aa6fe88acd5407ba41e69cbcce5664b7c38f8773